ImageEditor/src/test/kotlin/org/wordpress/android/imageeditor/crop/CropViewModelTest.kt

```diff
@@ -23,4 +26,14 @@ class CropViewModelTest {
         viewModel.onDoneMenuClicked()
         assertThat(viewModel.shouldCropAndSaveImage.value).isEqualTo(true)
     }
+
+    /*@Test
+    fun `file paths written to bundle `() {
```

Comment:
Tried setting up [nhaarman/mockito-kotlin](https://github.com/nhaarman/mockito-kotlin), to write a test case similar to this one. Encountered issues [parameter specified as non null is null](https://github.com/nhaarman/mockito-kotlin/wiki/Parameter-specified-as-non-null-is-null). Will update this part later.
https://github.com/wordpress-mobile/WordPress-Android/blob/abb29e724e7adae4af0104994d4a7a5e9399b783/WordPress/src/test/java/org/wordpress/android/ui/sitecreation/SiteCreationMainVMTest.kt#L219